src/upload.test.js

```diff
@@ -0,0 +1,104 @@
+const axios = require("axios");
+
+const { uploadDirectory, uploadFile } = require("./upload");
+const { uriSkynetPrefix } = require("./utils");
+
+jest.mock("axios");
+
+const portalUrl = "https://siasky.net";
+const skylink = "XABvi7JtJbQSMAcDwnUnmp2FKDPjg8_tTTFP4BwMSxVdEg";
+
+describe("uploadFile", () => {
+  const filename = "testdata/file1.txt";
+
+  axios.post.mockResolvedValue({ data: { skylink } });
```

Comment:
same comment as above

```suggestion
  beforeEach(() => {
    axios.post.mockResolvedValue({ data: { skylink } });
  });
```